frontend/src/campaign/campaignView.js

```diff
@@ -0,0 +1,80 @@
+import React from 'react';
+import {project_features_and_create_svg_paths} from "../common";
+import {MapPath} from "../UILibrary/components";
+// import * as PropTypes from 'prop-types';
+
+export class CampaignView extends  React.Component {
+    constructor(props) {
+        super(props);
+        this.state = {
+            map_data: null,
+            budgetData: null,
+        };
+        this.map_height = 800;
+        this.map_width = 800;
+    }
+
+    /**
+     * When this component is mounted to the DOM, get democracy score data from the server
+     */
+    async componentDidMount() {
+        try {
+            const map= await fetch('/api/state_map_geojson/ZAF/', {
```

Comment:
space around `=` (hmm... linter should have bitten you here; I'll take a look at why it didn't...)